modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -318,5 +349,13 @@ sub tweak_analyses {
     my $analyses_by_name = shift;
 
     $analyses_by_name->{'dump_full_fasta'}->{'-parameters'}->{'compara_db'} = '#ref_db#';
+    delete $analyses_by_name->{'datacheck_fan'}->{'-flow_into'}->{2};
+    delete $analyses_by_name->{'datacheck_fan_high_mem'}->{'-flow_into'}->{2};
+    $analyses_by_name->{'datacheck_factory'}->{'-parameters'}->{'compara_db'} = '#ref_db#';
+    $analyses_by_name->{'datacheck_fan'}->{'-parameters'}->{'compara_db'} = '#ref_db#';
+    $analyses_by_name->{'datacheck_fan'}->{'-parameters'}->{'old_server_uri'} = '#ref_db#';
+    $analyses_by_name->{'datacheck_fan'}->{'-flow_into'}->{0} = ['jira_ticket_creation'];
+    $analyses_by_name->{'datacheck_fan_high_mem'}->{'-flow_into'}->{0} = ['jira_ticket_creation'];
+    $analyses_by_name->{'store_results'}->{'-parameters'}->{'dbname'} = 'ensembl_compara_references';
```

Comment:
```suggestion
    $analyses_by_name->{'store_results'}->{'-parameters'}->{'dbname'} = '#ref_db#';
```

Reply:
The `dbname` has to explicitly be the `dbname` only - specific to the production side runnable - I hard-coded it here because I figure the name will not change for the reference db- but happy to include it as a parameter - to note the `dbname` cannot be seen elsewhere or it will need to be explicitly passed as `undef` to the other runnables to work :grimacing: 
Alternatively I could put a wrapper around the production runnable to collect the dbname - but this is only necessary to this specific pipeline because the dcs are running on the ref_db and not the pipelin_db.